src/LineageTree/lineageTree.py

```diff
@@ -2141,43 +2276,183 @@ def unordered_tree_edit_distance(
+        return uted(nodes1, adj1, nodes2, adj2, delta=delta_tmp) / norm(n1, n2)
+
+    def export_simple_nx_graph(self, start_time: int = None):
+        """
+        Creates a networkx tree graph. This function is to be used for visualization,
+        so only the start and the end of a branch are calculated, all cells in between are not taken into account.
+        Args:
+            start_time (int): From which timepoints are the graphs to be calculated.
+                                For example if start_time is 10, then all trees that begin
+                                on tp 10 or before are calculated.
+        returns:
+            G : list(networkx objects)
+            pos : list(dict(id:position))
+        """
+
+        mothers = list(self.time_nodes[0])
+        if start_time:
+            mothers = [
+                root for root in self.roots if self.time[root] <= start_time
+            ]
+        all_nodes = {}
+        all_edges = {}
+        for mom in mothers:
+            nodes = []
+            edges = []
+            cells = [mom]
+            while cells != []:
+                tmp_cells = []
+                for cell in cells:
+                    new_cells = self.successor.get(
+                        self.get_cycle(cell)[-1], []
+                    )
+                    nodes.append(self.get_cycle(cell)[-1])
+                    if cell != self.get_cycle(cell)[-1]:
+                        edges.append((cell, self.get_cycle(cell)[-1]))
+                    for new_cell in new_cells:
+                        nodes.append(new_cell)
+                        edges.append((self.get_cycle(cell)[-1], new_cell))
+                        tmp_cells.append(new_cell)
+                    cells = tmp_cells.copy()
+            all_nodes[mom] = nodes
+            all_edges[mom] = edges
+            G = {}
+        for i in range(len(mothers)):
+            G[i] = nx.DiGraph()
+            G[i].add_nodes_from(all_nodes[mothers[i]])
+            G[i].add_edges_from(all_edges[mothers[i]])
+        return G
+
+    def plot_all_lineages(
+        self, starting_point: int = None, nrows=2, figsize=(10, 15)
+    ):
+        """Plots all lineages.
+
+        Args:
+            starting_point (int, optional): From which timepoints are the graphs to be calculated.
+                                For example if start_time is 10, then all trees that begin
+                                on tp 10 or before are calculated. Defaults to None.
+            nrows (int):  How many rows of plots should be printed.
+        """
+        import matplotlib.pyplot as plt
+
+        from .utils import postions_of_nx
+
+        nrows = int(nrows)
+        if nrows < 1 or not nrows:
+            raise Warning("Number of rows  cannot be 0!")
+            nrows = 1
+
+        graphs = self.export_simple_nx_graph(start_time=starting_point)
+        ncols = int(len(graphs) // nrows) + (
+            len(graphs) // nrows != len(graphs) / nrows
```

Comment:
That's ugly, don't you know the `%` operator?
Also, if you really want to make it a one liner, don't add things that are not the same class:
```python
ncols = int(len(graphs) // nrows) + np.sign(len(graphs) % rows)
```